govcd/api_vcd_test.go

```diff
@@ -141,8 +141,10 @@ type TestConfig struct {
 		} `yaml:"nsxt_provider_vdc"`
 		Catalog struct {
 			Name                    string `yaml:"name,omitempty"`
+			NsxtBackedCatalogName   string `yaml:"nsxtBackedCatalogName,omitempty"`
```

Comment:
Should we also change `sample_govcd_test_config.yaml` to explain these new fields?

Reply:
Yes - checked it out from PR #495